packages/common/index.ts

```diff
@@ -317,6 +317,9 @@ export class KhouryDataParams {
   @IsInt()
   campus!: string;
 
+  @IsInt()
+  professor!: string;
+
```

Comment:
I'm not exactly sure how validation works shouldn't the `IsInt` decorator match up with a `number` type?

Reply:
I'm using line 318 as a precedent here -- it also uses `@IsInt()` on a string.